arnold/crud/read/step.py

```diff
@@ -56,7 +59,7 @@ def query_steps(
     Pagination can be enabled with <page_size> and <page_num> options.
     No pagination enabled by default.
     """
-    query_pipe = [step_filters.dict(exclude_none=True)]
+    query_pipe = [step_filters.model_dump(exclude_none=True)]
```

Comment:
Add type hint

Reply:
I will do the step refactoring in a separate PR - since it was a bit bigger than the others and more complex